spec/rubocop/cop/factory_bot/association_style_spec.rb

```diff
@@ -286,6 +286,24 @@ def inspected_source_filename
       end
     end
 
+    context 'when implicit association has nested factory and traits' do
```

Comment:
It’s not an association, it just happened that it was detected as such.

Reply:
1. There was a `factory inside a factory` spec already, so I moved this spec right underneath it and used the previous pattern as an example (factory name, trait, etc.)
2. I renamed it to match `when factory inside a factory with inline traits`